packages/opentelemetry-node/src/NodeTracerFactory.ts

```diff
@@ -14,36 +14,22 @@
  * limitations under the License.
  */
 
-import * as types from '@opentelemetry/types';
 import { NodeTracer } from './NodeTracer';
 import { NodeTracerConfig } from './config';
-import { SpanProcessor } from '@opentelemetry/tracing';
+import {
+  BasicTracer,
+  AbstractBasicTracerFactory,
+} from '@opentelemetry/tracing';
 
-export class NodeTracerFactory implements types.TracerFactory {
-  private readonly _tracers: Map<string, NodeTracer> = new Map();
-  private _spanProcessors: SpanProcessor[] = [];
+export class NodeTracerFactory extends AbstractBasicTracerFactory {
```

Comment:
It seems weird to me the `NodeTracerFactory` extends `AbstractBasicTracerFactory` when `NodeTracer` and `BasicTracer` are different